.vagrant/rgloader/loader.rb

```diff
@@ -0,0 +1,9 @@
+# This file loads the proper rgloader/loader.rb file that comes packaged
+# with Vagrant so that encoded files can properly run with Vagrant.
+
+if ENV["VAGRANT_INSTALLER_EMBEDDED_DIR"]
```

Comment:
We don't want to commit anything under the `.vagrant` directory, and it should be in a `.gitignore` file. Speaking of @avllee21, how's that coming along? 

@Imsiaocong, can you please remove this file from your pull request?